packages/nifti-volume-loader/src/enums/Events.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Cornerstone Nifti volume loader events
+ */
+enum Events {
+  /**
+   * Event fired when a Nifti volume is loaded completely
+   *
+   */
+  NIFTI_VOLUME_LOADED = 'CORNERSTONE_NIFTI_VOLUME_LOADED',
```

Comment:
Is it necessary to have different events from the dicom volume loader?  Really they should be the same events and just distinguish the type of data being loaded.  It might be necessary to have them be slightly different because of bulk loading, but ideally making them identical would be useful.

Reply:
I think so... , since there is no streaming happening?